templates/aws-vpc.template

```diff
@@ -44,15 +44,6 @@
                         "PrivateSubnetBTag3",
                         "VPCTenancy"
                     ]
-                },
-                {
-                    "Label": {
```

Comment:
We need to keep these two parameters: KeyPairName, NATInstanceType for backward compatibility. Otherwise, all quick starts that pass these two values to the nested VPCStack will break.